api/backtest_async.resolver.go

```
@@ -0,0 +1,198 @@
+	startDate, endDate time.Time,
+	samplingInterval time.Duration,
+) {
+	ctx := context.Background()
+	profile, endProfile := domain.NewProfile()
+	ctx = context.WithValue(ctx, domain.ContextProfileKey, profile)
+	defer endProfile()
+
+	lg := logger.New()
+
+	// Stage: loading price data
+	h.BacktestJobRepository.UpdateStage(jobID, domain.StageLoadingPriceData, 5)
```

golangci-lint (2.11.4)

[error] Error return value of `h.BacktestJobRepository.UpdateStage` is not checked

(errcheck)